pkgs/shells/bash/ble.sh/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, stdenv, git, fetchgit }:
+
+stdenv.mkDerivation {
```

Comment:
If there is no compilation, you can use `stdenvNoCC`

Reply:
There is a compilation.

Reply:
It seems no CC needed for compilation. I made a quick test of building it on stdenvNoCC and succeeded.